src/meta/name.rs

```diff
@@ -22,14 +21,18 @@ pub struct Name {
 
 impl Name {
     pub fn new(path: &Path, file_type: FileType) -> Self {
-        let name = match path.file_name() {
-            Some(name) => name.to_string_lossy().to_string(),
-            None => path.to_string_lossy().to_string(),
+        let full = path.to_string_lossy().to_string();
+
+        let name = if let Some(idx) = full.rfind('/') {
```

Comment:
This will cause issues in Windows. Even if it did not, this would be a bad idea.